packages/apollos-ui-media-player/src/MediaPlayer/PlayheadState.js

```diff
@@ -42,7 +42,8 @@ class ProviderWithoutApollo extends Component {
   }
 
   handleOnLoad = ({ duration }) => {
-    this.setState({ duration, isLoading: false });
+    if (duration > 0) this.setState({ duration, isLoading: false });
```

Comment:
If I'm reading this correctly this is just protecting against the same thing as line 10 which sets default value of `this.state.duration` to `1`. The problem is that `handleOnLoad` was passing its `duration` of `0` and thus `setState` set `duration` to `0`.  Let's document this so we can keep track of it better moving forward.

```suggestion
    // Only call setState if `duration > 0` to prevent division by zero in other places (`Seeker`, etc.)
    if (duration > 0) this.setState({ duration, isLoading: false });
```